### PR TITLE
make custom template for branding in header to show alt print logo

### PR DESCRIPTION
**Pull request by cjstevens78**

### CODE REVIEW

File: print-logo.jpg
Comment (Adnan-cds) [inline print-logo.jpg:1]: Are you sure you want to use a jpg file as a logo?  The other [logo](https://github.com/localgovdrupal/localgov_base_croydon/blob/1.x/logo.svg) is using an svg file.

File: templates/block/block--system-branding-block.html.twig [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * - site_name: Name for site as defined in Site information settings.
+ * - site_slogan: Slogan for site as defined in Site information settings.
+ */
+#}
+{% block content %}
+  {% if site_logo %}
+
+    <div class="branding">
+      <div class="branding__item branding__item--logo">
+        <img src="/themes/contrib/localgov_base_croydon/print-logo.jpg" alt="{{ site_name ? site_name : 'Home'|t }}" class="print__logo"/>
```
Comment (Adnan-cds) [inline templates/block/block--system-branding-block.html.twig]: Please try `{{ site_name ? site_name : 'Home'|t }}` instead.

Comment (cjstevens78) [reply templates/block/block--system-branding-block.html.twig]: @Adnan-cds thanks i have amended

File: print-logo.jpg
Comment (cjstevens78) [reply print-logo.jpg:1]: @Adnan-cds the svg wouldnt show in firefox so i went with a jpg - which i guess is better for print anyway.

Comment (Adnan-cds) [reply print-logo.jpg:1]: Okay, thanks.

File: templates/block/block--system-branding-block.html.twig (hunk 1 above)
Comment (Adnan-cds) [reply templates/block/block--system-branding-block.html.twig]: Thanks.

**Review by Adnan-cds — APPROVED** [review]
Thanks.

**Pull request merged**